composer.json

```diff
@@ -12,7 +12,7 @@
         }
     },
     "require": {
-        "php": ">=8.0",
```

Comment:
Can you add PHP7.4 to `.github/workflows/test.yml`?

Reply:
And update readme.md badge aswell :)

Reply:
@frankdekker done - I am doing this for the first time, so excuse me, if I am wrong

Reply:
@Housik Took a different approach, and added symfony/php80-polyfill instead for the `str_contains` method.
Created v1.1.0 release for this patch. Thanks for the PR.